libphal/phal_sbe.C

```diff
@@ -272,7 +271,92 @@ sbeError_t captureFFDC(struct pdbg_target *target)
 	// create ffdc file
 	tmpfile_t ffdcFile(bufPtr.getData(), ffdcLen);
 	return sbeError_t(exception::SBE_CMD_FAILED, ffdcFile.getFd(),
-			  ffdcFile.getPath().c_str());
+			  ffdcFile.getPath());
+}
+
+sbeError_t capturePOZFFDC(struct pdbg_target *target)
+{
+	assert(is_ody_ocmb_chip(target) == true);
```

Comment:
Only `assert(is_ody_ocmb_chip(target)` should be suffice. Just a suggestion.

Reply:
done